mscore/scoreview.h

```diff
@@ -66,6 +66,8 @@ enum class MagIdx : char;
 //---------------------------------------------------------
 
 struct SmoothPanSettings {
+    // these are all actually loaded from the loadFromPreferences method so don't change these initializations to change the default values,
```

Comment:
Then why is there a value in this file?

Reply:
Because initializing variables has no measurable cost and might prevent a bug or two in the future.